orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -41,6 +44,8 @@ class PackageInfo {
     this.extractVersion = extractVersion
     this.mapper = mapper
     this.buildDetailExtractor = new BuildDetailExtractor()
+
+    filePattern = Pattern.compile("${stage?.context?.package}.*\\.${packageType}")
```

Comment:
While it's probably the case in practice when the app is running, some of the other, preexisting tests pass `null` values which will cause exceptions to be thrown if the null-safe operators aren't present.